README.md

```diff
@@ -19,3 +19,32 @@ This repository contains machine-readable specifications for the auxillilary ser
 - [ ] Exhaustive RPC testing for Forest docker image.
 - [ ] Sync testing for PRs.
 - [ ] Export testing for PRs.
+
+## Running Forest In DigitalOcean
+
+## Architectural Implementation
+
+![Untitled Diagram drawio (8)](https://user-images.githubusercontent.com/47984109/215227510-dac5b8fb-8019-4388-a0e7-d5c432b95d70.png)
+
+The flow goes from Terraform for provisioning of the servers and Anisble to run all neccessary installations including forest.
+
+## Requirements 
+- RAM: 32GB
+- VCPU: 8
+- Startup Disk Size: 100 GB
+- Expected Total Disk Size: > 500 GB
+
+N/B: It's worth to note that some of the naming conventions can be changed to suit your deployment needs. 
+
+To test out the implementation, just access the server with appropiate `ssh` details in this manner `ssh -i ~/id_rsa chainsafe@ip_address`.
+
+Also, out of curiosity, to initiate the infrastructure, run the following:
+- Ensure to add all neccessary enviroment variables for AECCESS_ID and SECRET_KEY into the local machine via the `echo` command. This is needed for the backend and state configurations.
+- Run `make apply` in the terraform directory.
+- Run `ansible-playbook forest-docker-run.yaml` in the ansible directory.
+
+## Collaborators
+- [YOUR NAME HERE] - Feel free to contribute to the codebase by resolving any open issues, refactoring, adding new features, writing test cases or any other way to make the project better and helpful to the community. Feel free to fork and send pull requests.
+
+## Questions
+Feel free to reach out to the BlockOps team for more details on how to interact with the infrastructure if the need arises while in deployment.
```

Comment:
How to reach the BlockOps team?

Reply:
I'd change the comment; we should rather go with letting the fellow raise an issue or something